decidim-surveys/app/forms/decidim/surveys/admin/survey_question_form.rb

```diff
@@ -9,11 +9,19 @@ class SurveyQuestionForm < Decidim::Form
         attribute :id, String
         attribute :position, Integer
         attribute :mandatory, Boolean, default: false
+        attribute :question_type, String
+        attribute :answer_options, Array[SurveyQuestionAnswerOptionForm]
         attribute :deleted, Boolean, default: false
 
         translatable_attribute :body, String
 
         validates :position, numericality: { greater_than_or_equal_to: 0 }
+        validates :question_type, inclusion: { in: SurveyQuestion::TYPES }
+
+        # Private: answer options should be mapped manually.
+        def map_model(model)
+          self.answer_options = model.answer_options
```

Comment:
It is needed! I did some tests with Virtus stuff and the problem is answer_options is not a real model, just a serialized field and he doesn't know how to build the actual form field.